.travis.yml

```diff
@@ -2,17 +2,26 @@ sudo: false
 language: php
 
 php:
+  - 7.4snapshot
+  - 7.3
+  - 7.2
   - 7.1
   - 7.0
   - 5.6
-  - 5.4
-  - 5.5
+  - nightly
```

Comment:
Should we add `nightly`? Since phpUnit doesn't support it yet. In php 8 all deprecations will be removed so our code will not work anyway regarding the need to disable the deprecation warnings in phpunit.

Reply:
Yea I agree, I'll remove nightly